javascript-data-model/order-history.js

```diff
@@ -0,0 +1,61 @@
+var orderHistory = [
+  {
+    orderPlaced: 'August 4, 2020',
+    total: '$34.00',
+    shipTo: 'JS Masher',
+    orderNumber: '# 114-3941689-8772232',
+    deliveryDate: 'Aug 8, 2020',
+    handedDirectlyToResident: false,
+    items: [{
+      title: 'JavaScript for impatient programmers',
+      author: 'Rauschmayer, Dr. Axel',
+      price: '$31.55',
+      returnWindow: 'Sep 7, 2020'
+    }]
+  },
+  {
+    orderPlaced: 'July 19, 2020',
+    total: '$44.53',
+    shipTo: 'JS Masher',
+    orderNumber: '# 113-9984268-1280257',
+    deliveryDate: 'July 20, 2020',
+    handedDirectlyToResident: true,
+    items: [{
+      title: 'The Timeless Way of Building',
+      author: 'Alexander, Christopher',
+      price: '$41.33',
+      returnWindow: 'Aug 19, 2020'
+    }]
+  },
+  {
+    orderPlaced: 'July 4, 2020',
+    total: '$17.22',
+    shipTo: 'JS Masher',
+    orderNumber: '# 114-2875557-9059409',
+    deliveryDate: 'July 7, 2020',
+    handedDirectlyToResident: true,
+    items: [{
+      itemName: 'Gamecube Controller Adapter',
+      price: '$15.98',
+      returnWindow: 'Aug 5, 2020'
+    }]
+  },
+  {
+    orderPlaced: 'July 3, 2020',
+    total: '$138,93',
+    shipTo: 'JS Masher',
+    orderNumber: '# 113-2883177-2648248',
+    deliveryDate: 'July 5, 2020',
+    handedDirectlyToResident: false,
+    items: [{
+      itemName: 'GameCube Controller - Super Smash Bros. Edition (Nintendo Switch)',
+      price: '$94.95',
+      returnWindow: 'Aug 4, 2020'
+    }, {
+      title: 'The Art of Sql',
+      author: 'Faroult, Stephanie',
+      price: '$33.99',
+      returnWindow: 'Aug 4, 2020'
+    }]
```

Comment:
In these two objects, there is no indicator to tell whether this is a book that would have an author, or an item lacking an author. You might add an additional property named `type` to tell what this item is, such as a book, electronic, etc.